docs/osf/setup.rst

```diff
@@ -336,24 +369,26 @@ The following commands will install the requirements for add ons.
         invoke addon_requirements
 
 
-Starting up
------------
+Running The OSF
+***************
 
-Run your TokuMX process:
+Quick Start
+-----------
+To fully run the OSF, the following commands must be run.  Many of these programs will continue to run in order to log output to the console: this is normal!
 
+Therefore, run the following commands each in their own terminal windows, making sure to switch to the OSF virtual environment (and directory) each time:
     ::
 
-        invoke mongo
-
-Note -- TokuMX must be running in order to invoke the server. If the process stops it has failed. Try running  `mongod` for a more informative message. See below for common problems.
+            invoke mongo -d  # Runs mongod as a daemon
+            invoke mailserver
+            invoke rabbitmq
+            invoke celery_worker
+            invoke elasticsearch
+            invoke assets -dw
+            invoke server
```

Comment:
Doesn't have to go in this PR, but it would be useful to have a brief section on running these processes with launchctl. Processes run with launchctl get started in the background when your machine boots, so you don't have to run these commands all the time.

The basics:

1. Run `brew info elasticsearch`
2. Find the launchctl command in the output.
3. Run it.